core/src/main/java/org/elasticsearch/common/xcontent/XContentBuilder.java

```diff
@@ -83,6 +83,10 @@ public static XContentBuilder builder(XContent xContent, String[] filters) throw
         return new XContentBuilder(xContent, new BytesStreamOutput(), filters);
     }
 
+    public static XContentBuilder builder(XContent xContent, boolean inclusive, String[] filters) throws IOException {
```

Comment:
Can the order of arguments be consistent with XContent.createGenerator() method ?